public/index.html

```diff
@@ -35,14 +79,24 @@ <h2>
             <!-- I KNOW THIS MUST BE USED WITH LI,BUT IT 
             LOOKS COOLER ONLY IF YOU USE ONLY OL -->
             <ol>
```

Comment:
here instead of **ol** tag, you can use simple **div**
* Also don't forget to adjust styles in CSS 
ol{
    list-style-type: none;
    background-color: rgb(252, 239, 216);
}